src/test/java/org/candlepin/insights/controller/InventoryControllerTest.java

```diff
@@ -172,4 +174,51 @@ public void testNoneMacIsIgnored() {
         assertEquals(uuid, conduitFacts.getSubscriptionManagerId());
         assertTrue(conduitFacts.getMacAddresses().isEmpty());
     }
+
+    @Test
+    public void testIpAddressesCollected() {
+        String uuid = UUID.randomUUID().toString();
+        String systemUuid = UUID.randomUUID().toString();
+        Consumer consumer = new Consumer();
+        consumer.setUuid(uuid);
+        consumer.getFacts().put("net.interface.eth0.ipv4_address_list", "192.168.1.1, 1.2.3.4");
+        consumer.getFacts().put("net.interface.eth0.ipv4_address", "192.168.1.1");
+        consumer.getFacts().put("net.interface.lo.ipv4_address", "127.0.0.1");
+        consumer.getFacts().put("net.interface.eth0.ipv6_address.link", "fe80::2323:912a:177a:d8e6");
+
+        ConduitFacts conduitFacts = controller.getFactsFromConsumer(consumer);
```

Comment:
My only request is that we just make `extractIpAddresses` a protected method and just test it directly.  I like for tests to be as close to the code they are testing as possible and I don't see any major advantage in keeping `extractIpAddresses` private.

Reply:
Roger that!